frontend/src/metabase/collections/components/BaseItemsTable.styled.jsx

```diff
@@ -143,4 +154,10 @@ export const TBody = styled.tbody`
       }
     }
   }
+
+  tr:hover {
```

Comment:
Could we avoid a raw nesting selector like this? Could we replace it with a referenced to a styled component?

Reply:
Just for my own understanding, why would referencing a styled component itself be preferred here?

Reply:
Yeah, I'm not sure I can see a win here; we'd need to add an empty styled component for this